app/src/main/java/br/org/cesar/discordtime/stickysessions/navigation/router/Router.java

```diff
@@ -0,0 +1,35 @@
+package br.org.cesar.discordtime.stickysessions.navigation.router;
+
+import br.org.cesar.discordtime.stickysessions.navigation.exception.InvalidRouteException;
+import br.org.cesar.discordtime.stickysessions.navigation.repository.IRouteRepository;
+import br.org.cesar.discordtime.stickysessions.navigation.wrapper.IContextWrapper;
+
+public class Router implements IRouter {
+
+    private IContextWrapper mContext;
```

Comment:
I think is better to name it as `mContextWrapper` to avoid confusion with Android's context.